src/sentry/loader/browsersdkversion.py

```diff
@@ -4,11 +4,10 @@
 import re
 
 from django.conf import settings
-from pkg_resources import parse_version
+from packaging.version import Version
 
 import sentry
 from sentry.utils import json
-from sentry.utils.compat import map
```

Comment:
the only usage of `map` in this file is the version parsing line, where the value is immediately passed to `max` (making the extra `list` calls wasteful / redundant)

Reply:
Looking forward to you powertooling away `sentry.utils.compat` :3

Reply:
already started a few of them :D https://github.com/getsentry/sentry/pull/34557